src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -127,7 +127,16 @@ protected virtual void Initialize()
             }
 
             // Bindings may use name resolution, so provide this before reading the bindings. 
-            ScriptConfig.HostConfig.NameResolver = new NameResolver();
+            var nameResolver = new NameResolver();
+
+            var dashboardString = nameResolver.Resolve("AzureWebJobsDashboard");
```

Comment:
Actually you can use the ConnectionStringNames public static type that the core SDK exposes: ConnectionStringNames.Dashboard